react/features/base/connection/actions.web.js

```diff
@@ -28,6 +30,13 @@ export function connect() {
         // XXX For web based version we use conference initialization logic
         // from the old app (at the moment of writing).
         return APP.conference.init({ roomName: room }).then(() => {
+            // If during the conference initialization was defined that browser
+            // doesn't support WebRTC then we should define which route
+            // to render.
+            if (APP.unsupportedBrowser) {
```

Comment:
Yes, but if the room is `undefined` then router navigates to the WelcomePage. I used force appNavigate in order to define which router/component should be rendered for this user.